Week3/homework/2-step3.js

```diff
@@ -10,20 +10,18 @@ function threeFive(startIndex, stopIndex, threeCallback, fiveCallback) {
   console.log(values);
   for (let index = 0; index <= values.length; index++) {
     const value = values[index];
-    if (value % 3 === 0 && value % 5 === 0) { threeCallback(value), fiveCallback(value); }
-    else if (value % 3 === 0) { threeCallback(value); }
+    if (value % 3 === 0) { threeCallback(value); }
     else if (value % 5 === 0) { fiveCallback(value); }
 
   }
-
 }
 
 function sayThree(value) {
-  console.log(value + 'is divisible by three');
+  console.log(value + ' is divisible by three');
   return value;
 }
 function sayFive(value) {
-  console.log(value + 'is divisible by five');
+  console.log(value + ' is divisible by five');
   return value;
 }
```

Comment:
This is the output I get:

```
13:01 $ node 2-step3.js
[ 10, 11, 12, 13, 14, 15 ]
10 is divisible by five
12 is divisible by three
15 is divisible by three
```

The number 15 is also divisible by 5, but I don't see that in the output.